src/sentry/migrations/0561_backfill_new_notification_tables.py

```diff
@@ -217,7 +217,9 @@ def backfill_notification_settings(apps, schema_editor):
     NotificationSettingOption = apps.get_model("sentry", "NotificationSettingOption")
     NotificationSettingProvider = apps.get_model("sentry", "NotificationSettingProvider")
 
-    for setting in RangeQuerySetWrapperWithProgressBar(NotificationSetting.objects.all()):
+    for setting in RangeQuerySetWrapperWithProgressBar(
```

Comment:
@snigdhas don't put it in the query because now this becomes an unindexed query. Instead, do this check in memory.